community/bats/APKBUILD

```diff
@@ -31,6 +31,4 @@ package() {
 			"$pkgdir"/usr/share/man/man$i/bats.$i
 	done
```

Comment:
Consider switching to the `./install.sh` script so the `/usr/bin/bats` wrapper is installed correctly (a symlink does not work anymore)

e.g.
```bash
package() {
        cd "$builddir"
        ./install.sh "$pkgdir"/usr
}
```

I you like you can patch the script to install in `usr/libexec/bats/` instead of `usr/libexec/`

Or fix the package() phase to not create a symlink but instead install the `/usr/bin/bats` wrapper